src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -541,7 +535,6 @@ namespace EvaporativeFluidCoolers {
             thisEFC.HeatRejectCapNomCapSizingRatio = NumArray(8);
             thisEFC.HighSpeedStandardDesignCapacity = NumArray(9);
             thisEFC.LowSpeedStandardDesignCapacity = NumArray(10);
-            thisEFC.LowSpeedStandardDesignCapacitySizingFactor = NumArray(11);
```

Comment:
Several variables on this struct were never used, only assigned, so removed them.